.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Question
+about: 'Ask a question about the library. Or, join our Discord: https://discord.gg/FNudpBD'
+title: ''
+labels: question
+assignees: ''
```

Comment:
You can also go ahead and (optionally) add me as an assignee on the questions. One of the things I've been trying to keep up with are any questions that aren't answered in the user-guide/docs.

Reply:
Ok, done 👍🏼 thanks!
You can always un-assign yourself if you don't feel like answering 😬

Reply:
Sounds good! I might not always answer, but I'd like to be able to get notified for any interesting ones :)